src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+  </head>
+  <body>
+    <div class="page" data-qa="card">
+      <div class="logo page__img">
+        <img
+         class="logo__img"
+         src="images/imac.jpeg"
+         alt="logo"
+        >
+      </div>
+
+      <div class="page__title">
+        <h1 class="page__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      </div>
```

Comment:
Why do you need the div `here`?
```suggestion
        <h1 class="page__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Reply:
Delete redundant divs in all places in your code